citest/gcp_testing/gcp_error_predicates.py

```diff
@@ -75,7 +75,7 @@ def __init__(self, http_code=None, content_regex=None, **kwargs):
     self.__content_regex = content_regex
     super(HttpErrorPredicate, self).__init__(**kwargs)
 
-  def __call__(self, value):
+  def __call__(self, context, value):
```

Comment:
yes. ValuePredicates always take a context, value as the interface.
When I added context in I must have overlooked this class somehow.
This method will be used when you bind this predicate to an ObservationErrorPredicate to delegate error analysis to.